d2renderers/d2svg/fitToScreen.js

```diff
@@ -1,5 +1,10 @@
 window.addEventListener("DOMContentLoaded", () => {
   const svgEl = document.querySelector("svg");
+  const svgID = "d2-svg";
```

Comment:
I meant do this when generating the svg. Otherwise, our svg could be embedded inside another svg and this code will still run.